groups/OWNERS

```diff
@@ -0,0 +1,16 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+# root approval is restricted to members of k8s-infra-group-admins@
+# because they have the ability to manually run, troubleshoot, and undo
+# changes
+
+options:
+  no_parent_owners: true
+
+approvers:
```

Comment:
Should be fine now to have us as the approvers, but in the end we should add SC or TOC .

Reply:
In this folder, everyone should be able to able to modify the yaml files in the subfolders but not touch the groups folder